src/ceph.in

```diff
@@ -359,36 +359,56 @@ def wrap(s, width, indent):
 
     raise StopIteration
 
+
+def terminal_width():
+    if not sys.stdout.isatty():
+        return 80
+    try:
+        import shutil
+        sz = shutil.get_terminal_size((80, 20))
+        return sz.columns
+    except AttributeError:
+        import fcntl
+        import struct
+        import termios
+        arg = struct.pack('HHHH', 0, 0, 0, 0)
+        ret = fcntl.ioctl(sys.stdout.fileno(), termios.TIOCGWINSZ, arg)
```

Comment:
We should probably figure out a way to reuse ceph_daemon.Termsize here (maybe moving it out of ceph_daemon)